src/webapp/pages/form-page/useForm.ts

```diff
@@ -104,6 +111,7 @@ export function useForm(formType: FormType, id?: Id): State {
                             isIncidentManager: isIncidentManager,
                         }),
                     });
+                    setEntityData(formData);
```

Comment:
isn't formdata already set in setConfigurableForm ? why not reuse it?

Reply:
i'm updating `entityData` every time a form option is removed so i can get the `entityId` using the index. when i attempted to do this using `formData`, i wouldn't get the correct entityId and i would run into an error `onPrimaryButtonClick`